transforms/helpers/ast.ts

```diff
@@ -228,26 +219,27 @@ export function isEOPropertyForClassDecorator(
   );
 }
 
-export type EOClassDecoratorValue = Literal | ArrayExpression | Identifier;
+export type EOClassDecoratorValue =
+  | StringLiteral
+  | ArrayExpression
+  | Identifier;
 
-export function isEOClassDecoratorValue(
-  u: unknown
-): u is EOClassDecoratorValue {
+function isEOClassDecoratorValue(u: unknown): u is EOClassDecoratorValue {
   return (
-    isNode(u, 'Literal') ||
+    isNode(u, 'StringLiteral') ||
     isNode(u, 'ArrayExpression') ||
     isNode(u, 'Identifier')
```

Comment:
I'd probably make `isNode` take rest args so you could do:

```suggestion
    isNode(u, 'StringLiteral', 'ArrayExpression', 'Identifier')
```

But maybe the types are too annoying.

Reply:
I'll give it a try.

Reply:
The types were actually super easy. yay